gapipy/cache.py

```diff
@@ -1,13 +1,20 @@
 from time import time
 
-
 try:
     # Python 2
     import cPickle as pickle
 except ImportError:
     # Python 3
     import pickle as pickle
 
+try:
+    from django.conf import settings as django_settings
+
+    django_settings.configure()
+    from django.core.cache import cache as django_cache
```

Comment:
Might need to be something like that 👇 

```suggestion
    from django.core.cache import caches as django_caches
```

See below comment about `django_cache.__getitem__` not existing on the cache-proxy we get back from this import 👇